waterdata/templates/monitoring_location.html

```diff
@@ -6,6 +6,16 @@
 {% block extra_head_tags %}
     {% if status_code == 200 %}
         <meta name="description" content="{{ components.Description(stations[0].site_no, location_with_values, parm_grp_summary) }}">
+        <!-- tags for Facebook Open Graph -->
```

Comment:
I think we shouldn't be adding any of these meta tags if more than one site has been passed in. So this will need an if clause similar to the one on line 36

Reply:
Good point. I didn't think of that.